recipeers/ingredientes.py

```diff
@@ -1,5 +1,15 @@
-class Ingredientes:
-  def __init__(self, nombre: set[str], cantidad: float,  medida: str):
-    self.nombre = nombre
-    self.cantidad = cantidad
-    self.medida = medida
\ No newline at end of file
+class Ingrediente:
+    def __init__(self, nombre: str):
+        self.nombre = nombre
```

Comment:
¿Por qué habéis quitado la medida? ¿La medida es específica de cada ingrediente o no?